magalix-agent.yaml

```diff
@@ -64,8 +64,8 @@ rules:
 - apiGroups: ["", "extensions", "apps", "batch", "metrics.k8s.io", "networking.k8s.io"]
   resources: ["nodes", "nodes/stats", "nodes/metrics", "nodes/proxy", "namespaces", "pods", "limitranges", "deployments", "replicationcontrollers", "statefulsets", "daemonsets", "replicasets", "jobs", "cronjobs", "ingresses", "ingressclasses", "services", "networkpolicies"]
   verbs: ["get", "watch", "patch"]
-- apiGroups: ["", "rbac.authorization.k8s.io"]
-  resources: ["clusterrolebindings", "clusterroles", "roles", "rolebindings", "persistentvolumes", "persistentvolumeclaims"]
+- apiGroups: ["", "rbac.authorization.k8s.io", "storage.k8s.io"]
```

Comment:
I think the api group `" "` is redundant here

Reply:
isnt it needed for resources under `corev1` ?

Reply:
We already go those in the first role. I don't think we need to request it again.

Reply:
it's needed for the pv and pvc